File: rock-paper-scissors/rockpaperscissors.py
TEMP = 0

def on_gesture_shake():
    global TEMP
    TEMP = randint(0, 2)
    if TEMP == 0:
        basic.show_icon(IconNames.SCISSORS)
    elif TEMP == 1:
        basic.show_leds("""
            . . . . .
            # # # # #
            # . # . #
            # # # # #
            . . . . .
            """)
    else:
        basic.show_leds("""
            # # # # #
            # . . . #
            # . . . #
            # . . . #
            # # # # #
            """)
    basic.show_leds("""
        . . . . .
        . . . . .
        . . . . .
        . . . . .
        . . . . .
        """)
input.on_gesture(Gesture.SHAKE, on_gesture_shake)

File: rock-paper-scissors/rockpaperscissors.js
let TEMP = 0
input.onGesture(Gesture.Shake, function () {
    TEMP = randint(0, 2)
    if (TEMP == 0) {
        basic.showIcon(IconNames.Scissors)
    } else if (TEMP == 1) {
        basic.showLeds(`
            . . . . .
            # # # # #
            # . # . #
            # # # # #
            . . . . .
            `)
    } else {
        basic.showLeds(`
            # # # # #
            # . . . #
            # . . . #
            # . . . #
            # # # # #
            `)
    }
    basic.showLeds(`
        . . . . .
        . . . . .
        . . . . .
        . . . . .
        . . . . .
        `)
})
